searx/templates/oscar/preferences.html

```diff
@@ -194,6 +194,7 @@ <h3>{{ _('Engines') }}</h3>
                                 </tr>
                         {% for search_engine in engines_by_category[categ] %}
                             {% if not search_engine.private %}
+                            {% if minimum_safesearch > 0 and search_engine.safesearch or minimum_safesearch == 0 %}
```

Comment:
To ease template implementation and for consistency, why not move these conditions just bellow :
https://github.com/searx/searx/blob/da8b227044f45127f705f6ea94a72d368eea73bb/searx/webapp.py#L811-L812
?

(line 196 can move too, but it is not related to this PR).

Reply:
Agreed. I am closing this PR once we make a decision about the configuration.